### PR TITLE
fix

### PR DESCRIPTION
**Pull request by velrest**
- fix(ci): `uses` specified in steps implies an action instead of a workflow
- fix(ci): release workflow syntax

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -46,7 +46,6 @@ jobs:
 
   release-container-image:
     needs: release
-    runs-on: ubuntu-latest
```
Comment (luytena) [inline .github/workflows/release.yml:49]: No longer necessary because it's defined in the `release-container-image` workflow?

Comment (velrest) [reply .github/workflows/release.yml:49]: no `runs-on` is just a syntax error when a uses is defined on job level.

Comment (velrest) [reply .github/workflows/release.yml:49]: `runs-on` is only needed if `steps` is defined.

**Review by luytena — APPROVED** [review]

**Pull request merged**